config/acme/machines/config_batch.xml

```diff
@@ -77,6 +77,7 @@
       <arg flag="-q" name="$JOB_QUEUE"/>
       <arg flag="-W" name="$JOB_WALLCLOCK_TIME"/>
       <arg flag="-P" name="$PROJECT"/>
+      <arg flag="-u" name="$MAIL_USER"/>
```

Comment:
Remove these. Instead add a <batch_email_flag> attribute to the batch system.

Example
<batch_email_flag>u</batch_email_flag>